### PR TITLE
Task updating from oam

### PR DESCRIPTION
**Pull request by spawlows**
Porting changes in dr_prediction_z1_4xN_internal_avx() and av1_wiener_convolve_add_src_avx2() from OAM.

### CODE REVIEW

**Comment by hassount** [comment]
@ttrigui 
Any issues with this ?

File: Source/Lib/ASM_AVX2/wiener_convolve_avx2.c [hunk 1]
```diff
@@ -18,244 +18,285 @@
...
+ // horizontal filter output.
+
 void av1_wiener_convolve_add_src_avx2(const uint8_t *src, ptrdiff_t src_stride,
     uint8_t *dst, ptrdiff_t dst_stride,
-    const int16_t *filter_x, int32_t x_step_q4,
-    const int16_t *filter_y, int32_t y_step_q4,
-    int32_t w, int32_t h,
+    const int16_t *filter_x, int x_step_q4,
+    const int16_t *filter_y, int y_step_q4,
+    int w, int h,
```
Comment (ttrigui) [inline Source/Lib/ASM_AVX2/wiener_convolve_avx2.c]: please use int32_t instead of int. this applies to any new code.

**Comment by hassount** [comment]
@spawlows 
Could you please update this PR addressing the comments above so that we can merge it?

**Comment by spawlows** [comment]
Please Drop this pool request. I will prepare a new one with correct merges.

**Comment by hassount** [comment]
@spawlows 
Thank you.

**Pull request closed without merging**